include/tvm/meta_schedule/workload_registry.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#ifndef TVM_META_SCHEDULE_WORKLOAD_REGISTRY_H_
+#define TVM_META_SCHEDULE_WORKLOAD_REGISTRY_H_
+
+#include <tvm/ir/module.h>
+
+#include <unordered_map>
+#include <vector>
+
+namespace tvm {
+namespace meta_schedule {
+
+/*! \brief The class of workload tokens. */
+class WorkloadTokenNode : public runtime::Object {
+ public:
+  /*! \brief The workload's IRModule. */
+  IRModule mod;
+  /*! \brief The workload's structural hash. */
+  String shash;
+  /*! \brief The workload's token id. */
+  int64_t token_id_;
```

Comment:
Can you give more documentation on what the token id is? I can't tell it's use from this comment. I understand IR module and hash, but not token id.

Reply:
Per discussion with Cody, I am going to remove the workload registry thing, which can be confusing :-)